src/eei.cpp

```diff
@@ -1056,4 +1044,12 @@ namespace hera {
     }
     return true;
   }
+
+  //Safely charges gas for callDataCopy, returnDataCopy, codeCopy, and externalCodeCopy
```

Comment:
I meant to put the short description into the header and it should only say something along the lines of "safely (handling overflows) charge variable gas cost based on word size"